### ISSUE TITLE
no-restricted-disable rule isn't working with require-description rule

### ISSUE THREAD

**Issue by eppisapiafsl** [mysticatea/eslint-plugin-eslint-comments#78]
Hi

There is an edge case in the `no-restricted-disable` rule when the `require-description` is enable

The `no-restricted-disable` doesn't catch the disable line when there is a break line in the description

### When the description is single line, it work as expected

```
// eslint-disable-next-line eslint-comments/no-restricted-disable -- Catch the error
// eslint-disable-next-line react-hooks/exhaustive-deps  -- Needs to be called only on mounted component
```

### Description with multiline

```
/* eslint-disable-next-line react-hooks/exhaustive-deps  -- 
  This is not reported by the no-restricted-disable rule */
```

**Comment by MichaelDeBoey** [comment]
Hi @eppisapiafsl!

Since this repo is unmaintained, you might want to re-open this issue in the @eslint-community fork https://github.com/eslint-community/eslint-plugin-eslint-comments

For more info about why we created this organization, you can read https://eslint.org/blog/2023/03/announcing-eslint-community-org

**Comment by eppisapiafsl** [comment]
Thanks @MichaelDeBoey , will post in the community
